setup.py

```diff
@@ -1,48 +1,16 @@
-#!/usr/bin/env python3
-# Copyright (c) 2019-2024 the Eclipse BaSyx Authors
-#
-# This program and the accompanying materials are made available under the terms of the MIT License, available in
-# the LICENSE file of this project.
-#
-# SPDX-License-Identifier: MIT
+from setuptools import setup, find_packages
 
-import setuptools
-from basyx.aas import __version__
-
-with open("README.md", "r", encoding='utf-8') as fh:
-    long_description = fh.read()
-
-setuptools.setup(
+# Metadata is now in pyproject.toml, but we need this file for backwards compatibility
+setup(
     name="basyx-python-sdk",
-    version=__version__,
-    author="The Eclipse BaSyx Authors",
-    description="The Eclipse BaSyx Python SDK, an implementation of the Asset Administration Shell for Industry 4.0 "
-                "systems",
-    long_description=long_description,
-    long_description_content_type="text/markdown",
-    url="https://github.com/eclipse-basyx/basyx-python-sdk",
-    packages=setuptools.find_packages(exclude=["test", "test.*"]),
-    zip_safe=False,
+    packages=find_packages(exclude=["test", "test.*"]),
     package_data={
         "basyx": ["py.typed"],
         "basyx.aas.examples.data": ["TestFile.pdf"],
     },
-    classifiers=[
-        "Programming Language :: Python :: 3",
-        "License :: OSI Approved :: MIT License",
-        "Operating System :: OS Independent",
-        "Development Status :: 5 - Production/Stable",
-    ],
     entry_points={
         'console_scripts': [
-            "aas-compliance-check = basyx.aas.compliance_tool.cli:main"
+            "aas-compliance-check=basyx.aas.compliance_tool.cli:main"
         ]
```

Comment:
Seems like this should be removed here as well, since we moved the compliance tool to a different repository (https://github.com/rwth-iat/aas-compliance-tool)